@amimodel/compileC.m

```diff
@@ -123,40 +35,32 @@ function compileC(this)
     
     % generate hash for file and append debug string if we have an md5
     % file, check this hash against the contained hash
-    cppsrc = {'symbolic_functions','spline','edata','rdata','udata','tdata'};
+    cppsrc = {'amici', 'symbolic_functions','spline','edata','rdata','udata','tdata', 'amici_interface_matlab', 'amici_misc', 'amici_model_functions'};
```

Comment:
No, not anymore. I changed that here, so it is resolved at compile-time. The *wrap.h functions have the same interface. The implementation doesn't matter.